src/System.IO.Ports/src/Interop/Unix/Interop.Serial.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO.Ports;
+using System.Runtime.InteropServices;
+using Microsoft.Win32.SafeHandles;
+
+internal static partial class Interop
+{
+    internal static partial class Serial
+    {
+        [DllImport(Libraries.IOPortsNative, EntryPoint = "SerialPortOpen", SetLastError = true)]
```

Comment:
It seems that even for this function, we should follow the convention of other similar ones and name it with `Serial_` prefix. Maybe `Serial_OpenPort` ?

Reply:
Yes, we should prefix all of the exports with a name related to the library, e.g. everything in System.Native.so is prefixed with `SystemNative_`.